s3config.yaml.sample

```diff
@@ -19,7 +19,7 @@
 
 S3Config_Sections: [S3_SERVER_CONFIG, S3_AUTH_CONFIG, S3_MOTR_CONFIG, S3_THIRDPARTY_CONFIG, S3_VERSION_CONFIG]
 S3_SERVER_CONFIG:
-   S3_DAEMON_WORKING_DIR: "/var/motr/"                # S3 Daemon will run in the specified directory
+   S3_DAEMON_WORKING_DIR: "/var/log/seagate/motr/"      # S3 Daemon will run in the specified directory
```

Comment:
Doesn't it seems a little strange to run s3server daemon in a log directory !!!

Don't we have other option ?